OtripleS.Portal.Web/Brokers/API/ApiBroker.Teachers.cs

```diff
@@ -0,0 +1,19 @@
+// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OtripleS.Portal.Web.Models.Teachers;
+
+namespace OtripleS.Portal.Web.Brokers.API
+{
+      public partial class ApiBroker
+      {
+            private const string TeachersRelativeUrl = "api/teachers";
+
+            public async ValueTask<List<Teacher>> GetAllTeachers() =>
+                await this.GetAsync<List<Teacher>>(TeachersRelativeUrl);
+      }
+}
```

Comment:
Add one more line at the end. Also in other files where `new line at the end of the file` is missing.